From a4b9d2cabfe30d257a1518e2db69ad5069212f0e Mon Sep 17 00:00:00 2001
From: Alastair Hughes <hobbitalastair@gmail.com>
Date: Thu, 27 Oct 2016 16:48:49 +1300
Subject: [PATCH] Only regenerate the man page when required

Make the flex binary an order-only prerequisite, and add back the
prerequisites from before 7cfb440. This prevents rebuilding the man page
whenever the flex binary is rebuilt, which causes problems if help2man
is not installed and will never work when cross compiling.

Fixes #108.
---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 882843c..ad046d7 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -23,7 +23,7 @@ CLEANFILES = \
 	flex.vr \
 	flex.vrs
 
-flex.1: ../src/flex$(EXEEXT)
+flex.1: $(top_srcdir)/configure.ac $(top_srcdir)/src/flex.skl $(top_srcdir)/src/options.c $(top_srcdir)/src/options.h | ../src/flex$(EXEEXT)
 	$(help2man) --name='$(PACKAGE_NAME)' --section=1 \
 	--source='The Flex Project' --manual='Programming' \
 	--output=$@ $< \
-- 
2.10.1

